src/test/kotlin/org/rust/ide/intentions/AddWildcardArmIntentionTest.kt

```diff
@@ -10,24 +10,6 @@ package org.rust.ide.intentions
  */
 class AddWildcardArmIntentionTest : RsIntentionTestBase(AddWildcardArmIntention::class) {
 
-    fun `test empty match`() = doAvailableTest("""
```

Comment:
If we decided not to show the corresponding intention in this case, let's convert this test using `doUnavailableTest`